cloudevents/README.md

```diff
@@ -0,0 +1,29 @@
+# Cloudevents Controller for Tekton
+
+...
+
+### Motivation
+
+Send Cloudevents based on Tekton Resource Lifecycle.
+
+## Install
+
+Install and configure `ko`.
+
+```
+ko apply -f controller.yaml
+```
+
+This will build and install the controller on your cluster, in the namespace
+`tekton-pipelines`.
+
+## Example
+
+## Uninstall
+
+```
+$ kubectl delete namespace tekton-pipelines
+namespace "tekton-pipelines" deleted
```

Comment:
I think this is not a good idea. Tekton is installed in the `tekton-pipelines` namespace too, deleting the namespace is not a good option to uninstall.
```
ko delete -f config/
```